thrift/src/main/thrift/atoms/storyquestions.thrift

```diff
@@ -0,0 +1,27 @@
+namespace * contentatom.storyquestions
+namespace java com.gu.contentatom.thrift.atom.storyquestions
+#@namespace scala com.gu.contentatom.thrift.atom.storyquestions
+
+enum RelatedStoryLinkType {
+  TAG = 0,
+  STORY = 1,
+}
+
+struct Question {
+  1: required string questionId
+  2: required string questionText
+}
+
+struct QuestionSet {
+  1: required string questionSetId
+  2: required string questionSetTitle
+  3: required list<Question> questions
+}
+
+struct StoryQuestionsAtom {
+  1: required string relatedStoryId
+  2: required RelatedStoryLinkType relatedStoryLinkType 
```

Comment:
These questions can be related to a 'story' or a tag. Given we don't have the concept of a story yet this allows us to support both.